fe/fe-core/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -101,80 +101,83 @@
      *          60m     60 mins
      *          120s    120 seconds
      */
-    @ConfField public static String audit_log_dir = PaloFe.DORIS_HOME_DIR + "/log";
-    @ConfField public static int audit_log_roll_num = 90;
-    @ConfField public static String[] audit_log_modules = {"slow_query", "query", "load", "stream_load"};
-    @ConfField(mutable = true) public static long qe_slow_log_ms = 5000;
-    @ConfField public static String audit_log_roll_interval = "DAY";
-    @ConfField public static String audit_log_delete_age = "30d";
+    @ConfField
```

Comment:
Will this code style be more standardized? Do we need to write in this style in the future?

Reply:
This is done by IDEA automatically, maybe its good to format like this?